fragments/master.sh

```diff
@@ -25,6 +25,16 @@ if ( ! ps -ef | grep "/usr/bin/docker" | grep -v 'grep' &> /dev/null ); then
     exit 1
 fi
 
+RESTART_POLICY="no"
+START_MODE=$1
+
+if [[ -n ${START_MODE} && ${START_MODE} == "install" ]]; then
```

Comment:
this is currently done by atanas. so you don't have to worry about. was just a comment